gateway-service/src/main/resources/gateway-api-doc.json

```diff
@@ -1,131 +1,185 @@
                 "produces": [
-                    "*/*"
+                    "application/json"
                 ],
-                "parameters": [],
-                "responses": {
-                    "200": {
-                        "description": "OK",
+                "parameters": [
+                    {
+                        "name": "LoginRequest",
+                        "in": "body",
+                        "description": "User credentials to be authenticated",
                         "schema": {
-                            "$ref": "#/definitions/Authentication"
+                            "$ref": "#/definitions/LoginRequest"
+                        }
+                    }
+                ],
+                "security": [
+                    {
+                        "LoginBasicAuth": []
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "Authenticated",
+                        "headers": {
+                            "Set-Cookie": {
+                                "description": "Cookie named apimlAuthenticationToken contains authentication token",
+                                "type": "string"
+                            }
                         }
                     },
-                    "201": {
-                        "description": "Created"
+                    "400": {
+                        "description": "Bad request"
                     },
                     "401": {
                         "description": "Unauthorized"
                     },
-                    "403": {
-                        "description": "Forbidden"
-                    },
                     "404": {
                         "description": "Not Found"
+                    },
+                    "405": {
+                        "description": "Method Not Allowed"
                     }
                 }
             }
         },
-        "api/v1/auth/query": {
+        "/query": {
             "get": {
                 "tags": [
                     "Security"
                 ],
-                "summary": "Verify mainframe authentication token.",
-                "operationId": "verifyUsingGET",
+                "summary": "Validate authentication token.",
+                "description": "Use the `/query` API to validate token and retrieve the information associated within.\n\nThe query request requires the token in one of the following formats:\n  * Cookie named `apimlAuthenticationToken`.\n  * Bearer authentication \n    **Header example:** Authorization: Bearer *token*\n    \n The response is a JSON object which contains information associated with the token.    \n",
+                "operationId": "validateUsingGET",
                 "consumes": [
                     "application/json"
                 ],
                 "produces": [
-                    "*/*"
+                    "application/json"
```

Comment:
It shouldn't be defined ```produces``` for the GET method.

Reply:
Consumes removed.